gempy/plotting/plot.py

```diff
@@ -712,3 +712,51 @@ def plot_topology(geo_data, G, centroids, direction="y"):
     """
     PlotData2D.plot_topo_g(geo_data, G, centroids, direction=direction)
 
+
+def plot_stereonet(geo_data, litho=None, planes=True, poles=True, single_plots=False, show_density=False):
+    '''
+    Equal-area projection of the orientations dataframe using mplstereonet ('Schmidt Net')
+
```

Comment:
For the docstrings we are using "google style". Basically you have to write 
Args:
    - geo_data...
The best way to check if you did it properly is to set up pycharm for the google style (I think numpy style is the default) and looking in pycharm docs tab if the format is right.
It seems cherry picking but otherwise a couple of automation processes goes nuts